docs/pages/cloud/getting-started.mdx

```diff
@@ -3,20 +3,47 @@ title: Getting Started With Teleport Cloud
 description: Teleport hosted and managed by the Teleport team.
 ---
 
-## Step 1/3 Signup
+Here is a simple set of steps to access your cloud from the command line and easily add your first Server access.
 
-Sign up [here](https://goteleport.com/get-started/).
+## Step 1/5 Signup
 
-## Step 2/3 Install client tools
+Sign up for a cloud account [here](https://goteleport.com/get-started/).
+
+## Step 2/5 Access Web Console
+
+Access Web Console
+![Access Web Console](../../img/cloud/homepage-1.png)
+
+Select Add Server and press COPY to copy the script command
+![Add Server Script](../../img/cloud/addserver-2.png)
+
+## Step 3/5 Install Teleport Agent on Server
+
+Paste and run script
+
+![Past Install Script](../../img/cloud/runscript-3.png)
+
+Teleport Agent Installed
+![Completed Install](../../img/cloud/completed-script-4.png)
+
+
+## Step 4/5 Access Server
+
+Select close and Server can be accessed
+![Access Server](../../img/cloud/online-session-5.png)
+
+## Step 5/5 Access from Command Line
 
 Install client libraries:
 
 <Tabs>
   <TabItem label="Linux">
     ```bash
-    curl -O https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-    tar -xzf teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-    cd teleport
+    curl -O https://get.gravitational.com/teleport-ent-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+    # verify signature 
```

Comment:
it's now a different format: https://goteleport.com/docs/docs/best-practices/#varibles

`(=teleport.version=)`